spec/graphql/relay/mongo_relation_connection_spec.rb

```diff
@@ -482,4 +491,49 @@ def get_names(result, field_name)
     connection = GraphQL::Relay::BaseConnection.connection_for_nodes(relation)
     assert_equal GraphQL::Relay::MongoRelationConnection, connection
   end
+
+  describe "relations" do
+    let(:query_string) {%|
```

Comment:
This one is quick and dirty. Happy to iterate on a smaller query if you care, but could use some help in writing one.